docs/source/configuration/telemetry/exporters/tracing/datadog.mdx

```diff
@@ -251,3 +311,106 @@ telemetry:
 | `resource_mapping`    | See [config](#resource_mapping)     | A map of span names to attribute names. |
 | `span_metrics`        | See [config](#span_metrics)         | A map of span names to boolean.         |
 
+## Sampler configuration
+
+When using Datadog you will need to take into consideration if you want to use the Datadog APM view or rely on OTLP metrics to gain insight into the Router's performance.
+The Datadog APM view is driven by traces, and for this to be accurate 100% of requests must be sampled and sent to the Datadog agent.
+
+Tracing is expensive both in terms of APM costs but also Router performance, and typically you will want to set the `sampler` to a low value ion production environments.
+However, this will mean that the APM view will only show a small percentage of traces.
+
+Datadog Agent sampling is a mode where ALL traces are sent to the Datadog agent, but only a percentage of them are forwarded to Datadog. This makes the APM view accurate while keeping costs low
+at the cost of the Router having an effective sample rate of 100% under the hood.
+
+Use the following guidelines on how to configure the `sampler` and `preview_datadog_agent_sampling` to get the desired behavior:
+
+**I want the APM view to show metrics for 100% of traffic, and I am OK with the performance impact on the Router.**
```

Comment:
I like these bold, conversational-style "headings". So far in docs we haven't used these, but they're easy to understand. May reformat them in the future into lists.

```suggestion
**I want the APM view to show metrics for 100% of traffic, and I am OK with the performance impact on the router.**
```